### PR TITLE
Hono connection address suffix added - hono tenant id

### PR DESCRIPTION
**Pull request by abalarev**
Resolves https://github.com/eclipse-ditto/ditto/issues/1574 
Address resolving for hono connections is fixed to include a hono-tenant suffix. 
Assuming there would be no more than 1 connection between ditto and hono, the connectionId is used for this suffix. 
Therefore the connectionId must be set during creation to be the same as hono-tenant-id

### CODE REVIEW

File: documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-hono.md [hunk 1]
```diff
@@ -30,20 +33,23 @@ protocol identifier and the host name of `base-uri` to form the connection URI l
...
+ 
+Note: The {{connection:id}} will be replaced by the value of connectionId
 
 #### Source reply target
 Similar to source addresses, the reply target `address` is an alias as well. The single valid value for it is `command`. 
 It is resolved to Kafka topic/key like this:
-* `command` -> `hono.command/<thingId>` (&lt;thingId> is substituted by thing ID value).
+* `command` -> `{%raw%}hono.command.{{connection:id}}/<thingId>{%endraw%}` (&lt;thingId> is substituted by thing ID value).
+
+Note: The {{connection:id}} will be replaced by the value of connectionId
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-hono.md:52]:
```suggestion
Note: The `{%raw%}{{connection:id}}{%endraw%}` will be replaced by the value of the connection's actual connectionId.
```

This double curly braces also need to be escaped with raw/endraw.
And we should put inline code formatting around them with single back ticks.

**Review by thjaeckle — CHANGES_REQUESTED** [review]
The documentation needs additional escaping - apart from that: LGTM :)

File: documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-hono.md [hunk 2]
```diff
@@ -30,20 +33,23 @@ protocol identifier and the host name of `base-uri` to form the connection URI l
...
 Valid source addresses (aliases) are `event`, `telemetry` and `command_response`. 
 Runtime, these are resolved as following:
-* `event` -> `hono.event`
-* `telemetry` -> `hono.telemetry`
-* `command_response` -> `hono.command_response`
+* `event` -> `{%raw%}hono.event.{{connection:id}}{%endraw%}`
+* `telemetry` -> `{%raw%}hono.telemetry.{{connection:id}}{%endraw%}`
+* `command_response` -> `{%raw%}hono.command_response.{{connection:id}}{%endraw%}`
+ 
+Note: The {{connection:id}} will be replaced by the value of connectionId
```
[the remark this answers is not in the document]
Comment (abalarev) [reply documentation/src/main/resources/pages/ditto/connectivity-protocol-bindings-hono.md]: Thanks, I've fixed it.

**Review by thjaeckle — APPROVED** [review]

**Comment by thjaeckle** [comment]
LGTM, thx @abalarev for addressing this so quickly 👍

**Pull request merged**